conduct/COC_POLICY.md

```diff
@@ -120,6 +120,7 @@ Projects are expected to participate in good faith in the escalation process and
 ## Appeals
 
 Harmed individuals and accused individuals have thirty (30) days after a response was provided to appeal to the CoC Team a decision made in response to an incident report. Decisions may only be appealed on the grounds that the documented process was not properly followed or that it did not meet the requirements of this policy. If the CoC Team finds that the appeal has merit, it may decide to override the decision that was previously made.
+The decision of the CoC team must be communicated to all parties involved within thirty (30) days.
```

Comment:
Is there some way we can make this and the other use of "all parties" more specific? Who exactly will be informed? And if we can't do that, might it be better to provide a little more discretion by using "relevant parties" or something like that?